src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -4,8 +4,8 @@ import * as React from 'react'
 import 'react-dropdown/style.css'
 
 import CityModel from '../../../modules/endpoint/models/CityModel'
-import type { FeedbackDataType } from '../../../modules/endpoint/FeedbackEndpoint'
-import FeedbackEndpoint, {
+import type { ParamsType as FeedbackParamsType } from '../../../modules/endpoint/endpoints/feedback'
+import feedback, {
```

Comment:
I think naming this `feedbackEndpoint` makes sense, we also use this kind of terminology in the routes (e.g. `categoriesEndpoint` in the `categories` route), but you could also leave it like this of course. If you should change it, there is also an occurance in `SearchFeedback`